content/sensu-go/5.21/reference/rbac.md

```diff
@@ -360,6 +367,7 @@ spec:
   - ops
   - dev
   password: USER_PASSWORD
+  password_hash: $5f$14$.brXRviMZpbaleSq9kjoUuwm67V/s4IziOLGHjEqxJbzPsreQAyNm
```

Comment:
Maybe we should mention that you only need to set `password` or `password_hash`, and you should prefer the later. WDYT?

Reply:
Yes, I think that's a good idea.